README.md

```diff
@@ -1,40 +1,94 @@
 Confluent's Apache Kafka client for Python
 ==========================================
 
+Confluent's Kafka client for Python wraps the librdkafka C library, providing
+full Kafka protocol support at great performance and reliability.
 
-Prerequisites
-===============
+The Python bindings provides a high-level Producer and Consumer with support
+for the balanced consumer groups of Apache Kafka 0.9.
 
-    librdkafka >=0.9.1 (or master>=2016-04-13)
-    py.test  (pip install pytest)
+See the [API documentation](http://docs.confluent.io/3.0.0/clients/confluent-kafka-python/index.html) for more info.
```

Comment:
If you want this link versioned, that's fine. You can also use http://docs.confluent.io/current/clients/confluent-kafka-python/index.html if you want something that doesn't need updating w/ each version